GOFevaluation/utils.py

```diff
@@ -390,10 +392,11 @@ def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
         bin_edges[0] = xlim[0]
         bin_edges[-1] = xlim[1]
         for low, high in zip(bin_edges[:-1], bin_edges[1:]):
-            rec = Rectangle((low, 0), high - low, 1,
-                            facecolor=cmap(norm(ns[i])),
-                            **kwargs)
-            ax.add_patch(rec)
+            ax.axvspan(low,
+                       high,
+                       facecolor=cmap(norm(ns[i])),
+                       **kwargs,
+                       )
```

Comment:
just fixing an old bug here: We should put `edgecolor = kwargs.pop('edgecolor', 'k')` further up in the code and then pass this also to the axvspan (as we do it in the 2d case with the rectangle).

Reply:
done!